src/main/resources/application.properties

```diff
@@ -0,0 +1,40 @@
+############################################
+###       APPLICATION PARAMETERS         ###
+############################################
+
+spring.application.name=blacklist
+server.address=0.0.0.0
+server.port=8444
+
+# Database connection (mandatory)
+# Change the server timezone if neccessary
+spring.datasource.url=jdbc:mysql://127.0.0.1:3306/ah_blacklist?serverTimezone=UTC
+spring.datasource.username=blacklist
+spring.datasource.password=bl@ckl1st
```

Comment:
please use a password generator to create a secure password. Users tend to not change this password at all.